apps/website/src/utils/image-converter.ts

```diff
@@ -0,0 +1,91 @@
+import type { Dispatch, SetStateAction } from "react";
+
+async function convertHeicToJpeg(file: File) {
+  const heic2any = (await import("heic2any")).default;
+  const blob = await heic2any({
+    blob: file,
+    toType: "image/jpeg",
+    quality: 0.9,
+  });
+  return new File([blob as Blob], file.name.replace(/\.[^.]+$/, "jpeg"), {
+    type: "image/jpeg",
+  });
+}
+
+async function convertAvifToJpeg(file: File) {
+  const convertedFile: File = await new Promise((resolve, reject) => {
+    const img = new Image();
+    const url = URL.createObjectURL(file);
+
+    img.onload = () => {
+      const canvas = document.createElement("canvas");
+      canvas.width = img.width;
+      canvas.height = img.height;
+      const ctx = canvas.getContext("2d");
+      if (!ctx) {
+        URL.revokeObjectURL(url);
+        reject(new Error("Canvas context not available"));
+        return;
+      }
+      ctx.drawImage(img, 0, 0);
+      canvas.toBlob(
+        (blob) => {
+          URL.revokeObjectURL(url);
+          if (!blob) {
+            reject(new Error("Conversion to JPEG failed"));
+            return;
+          }
+          resolve(
+            new File([blob], file.name.replace(/\.avif$/i, ".jpg"), {
```

Comment:
```suggestion
            new File([blob], file.name.replace(/\.avif$/i, ".jpeg"), {
```
Consistency with the `File` constructor further up